README.md

```diff
@@ -8,4 +8,8 @@ Some of the images are given here -
 
 "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQKgAjahWEIs10LFxpXY1Q3ihF83q1kd_qNFw&usqp=CAU"
 
-"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSyv_FXBv-r-NHFVZBuc-WxLAhPjl36SFsVQw&usqp=CAU"
\ No newline at end of file
+"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSyv_FXBv-r-NHFVZBuc-WxLAhPjl36SFsVQw&usqp=CAU"
+
+"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSutU6Jmnu7Inir1tI-kr49tcwspmBfJlKEbg&usqp=CAU"
+
+"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS8bR5q-JpQNkrDlA-8clXZyaZltB-WcxsGq66rc5bplyY7vfvUKAiuv0kUyTId-4KRc4c&usqp=CAU"
```

Comment:
Consider using a Markdown image syntax to display images directly in the README, instead of just providing the URLs. [medium]